src/wheel/cli/convert.py

```diff
@@ -72,13 +72,16 @@ def convert_pkg_info(pkginfo: str, metadata: Message):
 
         if key_lower == "description":
             description_lines = value.splitlines()
-            value = "\n".join(
-                (
-                    description_lines[0].lstrip(),
-                    dedent("\n".join(description_lines[1:])),
-                    "\n",
+            if len(description_lines) == 0:
+                value = "\n"
+            else:
+                value = "\n".join(
+                    (
+                        description_lines[0].lstrip(),
+                        dedent("\n".join(description_lines[1:])),
+                        "\n",
+                    )
```

Comment:
This is just my personal preference.
```suggestion
            if description_lines:
                value = "\n".join(
                    (
                        description_lines[0].lstrip(),
                        dedent("\n".join(description_lines[1:])),
                        "\n",
                    )
            else:
                value = "\n"
```